snippets/MultiselectCheckbox.md

```diff
@@ -64,8 +64,5 @@ ReactDOM.render(
 );
 ```
 
-#### Notes:
-
-- The checkbox input in this case is kept as `readOnly` since click events are handled for the item ( label + input ) and not the `input` alone.
-
-expertise: 0
+<!-- tags: input,state,array -->
+<!-- expertise: 1 -->
```

Comment:
@Chalarangelo I'm not sure how to decide on a particular expertise level for a snippet. Can you help me out with that? If you're on gitter we can continue the conversation there too so that we keep the PR clean!

Reply:
I usually just go with my gut, it's a bit subjective. I tagged this one as `1` instead of `0` as it might not be the easiest snippet to understand for an absolute beginner and it features a few techniques (such as `readonly` checkboxes) that might need a little bit of getting used to. 

Generally speaking, tag the expertise you feel that the snippet requires. Me and the rest of the team check every PR carefully and we evaluate these things, so don't worry too much about it as we can update them as we see fit. If the community thinks otherwise, we will change expertise levels to match what people think is appropriate for each snippet.